src/registrar/templates/django/admin/includes/domain_information_fieldset.html

```diff
@@ -0,0 +1,2 @@
+{% extends "django/admin/includes/detail_table_fieldset.html" %}
+{# Stubbed file for future expansion #}
```

Comment:
(Q): Why?

Reply:
I disagree. Make a note of this in case this is actually needed in the future, but keep in mind that the thinking from product is that we only invest what is necessary/has huge ROI in DJA and we save our time and energy for building a bespoke solution (in other words, that expansion you're assuming will happen is up in the air)